src/classes/TriggerHandler.cls

```diff
@@ -130,15 +131,17 @@ public virtual class TriggerHandler {
 
   // increment the loop count
   @TestVisible
-  private void addToLoopCount() {
```

Comment:
Hi I am new to salesforce and I am implementing this functionality like we can control the number of times a trigger executes. I have seen one method setMaxLoopCount(1), but it is not working. Can you please help me getting code for this method.